dev/MRTCore/mrt/mrm/mrmmin/mrmmin.vcxproj

```diff
@@ -376,14 +376,19 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\WindowsAppSDK_Insights\WindowsAppSDKInsightsLib\WindowsAppSDKInsightsLib.vcxproj">
+      <Project>{4243b8da-bdb5-4d65-b1d5-eeed375fad13}</Project>
+    </ProjectReference>
+  </ItemGroup>
```

Comment:
why do we need this?

Reply:
The Insights lib and header are needed to log telemetry.

Reply:
This same question is being answered in another comment. So I'll close this dupe.